scripts/make_api_rst.py

```diff
@@ -36,6 +36,55 @@
 Welcome to the QGIS Python API documentation project
 ==============================================================
 
+Introduction
+------------
+
+`QGIS <https://qgis.org>`_ is a user friendly Open Source Geographic Information System (GIS) that runs on Linux, Unix, Mac OSX, and Windows.
+QGIS supports vector, raster, and database formats. QGIS is licensed under the GNU General Public License.
+QGIS lets you browse and create map data on your computer. It supports many common spatial data formats (e.g. ESRI ShapeFile, GeoPackage, geotiff).
+QGIS supports plugins to do things like display tracks from your GPS. QGIS is Open Source software and its free of cost (download here).
+We welcome contributions from our user community in the form of code contributions, bug fixes, bug reports, contributed documentation,
+advocacy and supporting other users on our mailing lists and forums. Financial contributions are also welcome.
+
+QGIS source code is available at https://github.com/qgis/QGIS.
+There is also a `C++ version of the API documentation <https://api.qgis.org>`_ available.
```

Comment:
https://api.qgis.org does not open the C++ API doc here, I need to use https://api.qgis.org/api/

Reply:
Indeed. I did not test the URL, but still a weird behavior IMHO
```suggestion
There is also a `C++ version of the API documentation <https://api.qgis.org/api>`_ available.
```

Reply:
> but still a weird behavior IMHO

Could be fixed on server side in apache/nginx conf